api/.gitignore

```diff
@@ -1,2 +1 @@
 node_modules
-.env
```

Comment:
The `.env` files do not have to get pushed to git, as they contain sensitive information about your usernames and password. Add this line back such that git ignores that file, but still keeps it on your local machine. 😉